.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,66 @@
+<!--
+  Make sure you have read CONTRIBUTING.md completely before you file a new
+  issue! 
+
+  If possible, try to determine if the bug is actually part of the Swift driver,
+  or if the issue is actually from `libmongoc` or `libbson`. If so, you should
+  file the issue with the representative projects.
+-->
+
+## Information
+
+### Swift version
+
+What does this command give you?
+```
+$ swift --version
+```
+
+### Operating system
+
+What does this command give you?
+```
+$ uname -a
+```
+
+### Driver version
+
+What does this command give you?
+```
+$ cat Package.swift # Only applies if you are using Swift package manager
+```
+*or*
+```
+$ cat Podfile # Only applies if you are using Cocoapods
+```
+
+### How did you install `libmongoc` and `libbson` on your system
+Did you use `brew`? Did you install them manually? etc.
+
+### Version of `libmongoc` and `libbson`
+
+What does this command give you?
+```
+$ brew list --versions mongo-c-driver # Only applies if you installed via brew
+```
+
+## What is the problem? 
+
+**BE SPECIFIC**:
+* What is the _expected_ behavior and what is _actually_ happening?
+* Do you have any particular output that demonstrates this problem?
+* Do you have any ideas on _why_ this may be happening that could give us a
+clue in the right direction?
+* Did this issue arise out of nowhere, after an update (of the driver? of
+Swift?)? 
```

Comment:
I might suggest replacing the parenthetical with: "(of the driver, server, and/or Swift)"

Reply:
I don't see it mentioned elsewhere in the new issue template, but it probably makes sense to request the server version/topology when applicable.

Reply:
@jmikola Great idea about the server topology. Glad to have some more drivers-team perspective here, because I don't really have any experience debugging user issues with drivers, and did not think of asking for topology. Thanks.

Your other comment about the parenthetical is done.